net/node/link.go

```diff
@@ -305,8 +305,12 @@ func TLSDial(nodeAddr string) (net.Conn, error) {
 func (node *node) Tx(buf []byte) {
 	log.Debugf("TX buf length: %d\n%x", len(buf), buf)
 
+	if node.GetState() == INACTIVITY {
```

Comment:
How can we recover from inactivity to other status in this case?

Reply:
info update logic will call fetchRetryNodeFromNeiborList() to reconnect INACTIVITY node, which could recover the node`s state.